sakura_core/view/CEditView.cpp

```diff
@@ -1915,6 +1912,7 @@ bool CEditView::GetSelectedData(
 		}
 	}
 	else{
+		CEol appendEol(neweol);
 		cmemBuf->SetString(L"");
```

Comment:
```suggestion
		cmemBuf->Clear();
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17